app/models/match_decision_reasons/case_contact_assigns_manager_decline.rb

```diff
@@ -0,0 +1,13 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/boston-cas/blob/production/LICENSE.md
+###
+
+module MatchDecisionReasons
+  class CaseContactAssignsManagerDecline < Base
+    def title
+      "#{_('Stabilization Service Provider Nine')} Decline"
```

Comment:
This says `Nine`, but is used in `EightAssignManager` below? Do we need separate classes `EightCaseContactAssignsManagerDecline` and `NineCaseContactAssignsManagerDecline` ?

Reply:
Another idea is to pass the route name into the constructor...
```suggestion
      actor_name = "Stabilization Service Provider #{@route_name}"
      "#{_(actor_name)} Decline"
```

Reply:
So, I ended up fighting re-use of decisions as I went and ended up leaning toward massive duplication to avoid problems, but I don't think we need that for the decision reasons since those are setup differently.  I was going to implement your idea, though it turns out that'd be very difficult, and given that two routes have different contact roles playing the parts, I think we do need one specifically for each route.